joomla/jissues
# New vagrant box

Pull Request for Issue #918.

#### Summary of Changes

Added a new [contrib-jessie64](https://atlas.hashicorp.com/debian/boxes/contrib-jessie64/versions/8.6.2) vagrant box. Tested with the latest Vagrant 1.9.0 and Virtualbox 5.1.10.
Please note that I was testing it on Windows 10. But there should be no problem on other environments.

> vagrant@contrib-jessie:~$ php -v
> PHP 5.6.27-0+deb8u1 (cli) (built: Oct 15 2016 15:53:28)
> Copyright (c) 1997-2016 The PHP Group
> Zend Engine v2.6.0, Copyright (c) 1998-2016 Zend Technologies
>     with Zend OPcache v7.0.6-dev, Copyright (c) 1999-2016, by Zend Technologies
>     with Xdebug v2.2.5, Copyright (c) 2002-2014, by Derick Rethans
> 

> vagrant@contrib-jessie:~$ sudo apache2 -v
> Server version: Apache/2.4.10 (Debian)
> Server built:   Sep 15 2016 20:44:43

> vagrant@contrib-jessie:~$ sudo mysql -v
> Welcome to the MySQL monitor.  Commands end with ; or \g.
> Your MySQL connection id is 43
> Server version: 5.5.53-0+deb8u1 (Debian)

#### Testing Instructions
1. Pull the branch.
2. Run `vagrant up`. Box should be downloaded and provisioned without errors. 
3. When the box is ready, open http://localhost:2345/. Something should be displayed :)
